ArchitectureSpec.md

```diff
@@ -0,0 +1,74 @@
+# Vision Input Team:
+Handles image, video, and camera input and generates arrays of RGB or depth data
+## Responsibilities
+- Finding Camera parameters
+- Field of View (FoV)
+- Handle multiple cameras
+- Synchronization
+## Class structure
+4 ImageGenerator classes
+- `DepthDataGenerator (single frame depth filename, still image filename)`
+- `DepthLiveGenerator (Realsense camera port int)`
+- `ImageGenerator (single image frame filename)`
+- `VideoFileGenerator(filename)`
+- `VideoLiveGenerator(camera port input int)`
+## API
+### get_horizontal_fov()
+Returns float degrees
```

Comment:
Can we change the return type to radians for the fov methods so the other teams can use it to calculate the focal length?

Reply:
Sure, but we should probably change all our angle units to radians in that case. @michelle-pan @gargideshpande any thoughts?

Reply:
I think the rotation angles would be better in degrees because the range of values is more intuitive. If we want all the angle units to be uniform, could we keep the FoV in degrees and just convert it if necessary?

Reply:
Yes, we can just keep it in degrees then.